lib/features/pokemon_overview/widget/pokemon_card.dart

```diff
@@ -9,23 +10,32 @@ class PokemonCard extends StatelessWidget {
   const PokemonCard({
     super.key,
     required this.pokemon,
+    // required this.onTap,
   });
 
   final Pokemon pokemon;
+  // final Function(String) onTap;
 
   @override
   Widget build(BuildContext context) {
-    return Card(
-      child: Column(
-        mainAxisAlignment: MainAxisAlignment.center,
-        children: [
-          FadeInImage(
-            placeholder: const AssetImage(cardTransparentPlaceholder),
-            image: NetworkImage(pokemon.url.toPokemonImageUrl),
-            width: cardImageSize,
-          ),
-          BodySmallText(text: pokemon.name.capitalize())
-        ],
+    return GestureDetector(
+      onTap: () => Navigator.pushNamed(
```

Comment:
kindly put this in a separate private function to make more readable what's this for 🙂